alexrallen/sonic-buildimage
# [device][mellanox] Add 2x100G and 4x50G breakout modes to MSN4410 platform

#### Why I did it

The MSN4410 platform was missing 2x100G and 4x50G supported breakout modes in `platform.json`

#### How I did it

Added the aforementioned breakout modes to `platform.json`

#### How to verify it

Run `show interface breakout` on a MSN4410 and verify that 2x100G and 4x50G are listed in the supported breakout modes for ports 1 - 24. 

#### Which release branch to backport (provide reason below if selected)


#### Description for the changelog
[device][mellanox] Add 2x100G and 4x50G breakout modes to MSN4410 platform


#### A picture of a cute animal (not mandatory but encouraged)
![duckling](https://user-images.githubusercontent.com/5898707/127694595-d838111a-3954-4dd5-9abd-c43c946bd0a5.jpg)
